libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/EHorizonObjectProvider.kt

```diff
@@ -0,0 +1,26 @@
+package com.mapbox.navigation.core.trip.model.eh
+
+import androidx.annotation.StringDef
+
+/**
+ * RoadObject provider
+ */
+object EHorizonObjectProvider {
+    /**
+     * Road object was provided via Mapbox services
+     */
+    const val MAPBOX = "MAPBOX"
+
+    /**
+     * Road object was added by user
+     * (via `mapboxNavigation.getEHorizonObjectsStore.addCustomRoadObject()`)
```

Comment:
```suggestion
     * (via [EHorizonObjectsStore.addCustomRoadObject])
```
Use KDoc links, but remember to add an `import` as well.

Reply:
Same here and everywhere else.

Reply:
done